src/remotelog.lua

```diff
@@ -62,6 +68,15 @@ function M.set_client_name(log_client_name)
     M.log_client_name = log_client_name
 end
 
+---
+-- Set the fallback strategy.
+--
+-- @param fallback_strategy what to do if the remote listener connection cannot be established (default: console log)
+--
+function M.set_fallback_strategy(strategy)
```

Comment:
Is it necessary to have a setter? You can already configure this via `init`. Is it necessary to change the fallback strategy at runtime?
Keep API minimal if possible.

Reply:
Removed setter.